style.css

```diff
@@ -1,39 +1,129 @@
-@import url('https://fonts.googleapis.com/css2?family=Inter');
-body{
-    box-sizing: border-box;
+@import url("https://fonts.googleapis.com/css2?family=cyrillic Roboto");
+@import url("https://fonts.googleapis.com/css2?family=Poppins");
+@import url("https://fonts.googleapis.com/css2?family=Crete Round");
+
+body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  background-repeat: no-repeat;
+}
+
+a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.container {
+  border: 1px white;
+  width: 100vw;
+  height: 640px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
- [ ] Your design has some issues that need to be addressed before proceeding. some of these requirements are OPTIONAL and some of these are mandatory as they are introduced in this PR. Kindly consider fixing them all.
- [Mandatory] Your social icons have a border which is not there in the figma design and it also got introduced in this latest PR so kindly fix that.
- [Mandatory] And also try to give the exact space and margin between these icons as in the figma design.
- [Mandatory] The illustration background-image is not in the right position and that is due to the unresponsive state of your design. so kindly consider fixing that.
- [OPTIONAL] Your main title and subtitle in the headline section are bolder than the figma design. kindly try to make them similar in boldness.